docs/proposals/job.md

```diff
@@ -0,0 +1,145 @@
+# Job Controller
+
+## Abstract
+A proposal for implementing a new controller - Job controller - which will be responsible
+for managing pod(s) that require to run-once to a completion, in contrast to what
```

Comment:
I would say "run-once to a completion even if the machine the pod is running on fails" to make it clearer why ReplicationController with restart policy of RestartOnFailure isn't sufficient for what you want.